### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Import ExtensionModify Columns_AI4D: The controls in the table have no name set.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4176]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS;-A11yTCS;-A11ySev2;-BM_SQLAzureDataStudio_Oct2020; -SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;- WCAG4.1.2;-Benchmark;-AI4D;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW use the right-click key on the keyboard on it and select import wizard.
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Launch accessibility insights for desktop and hover over checkbox in column 3 row 1

**Actual:**
On hovering over the checkbox in column 3 row 1 there is no name set for it.
Same issue for the controls in row 1 column 4 and row 2 column 1,2,3,4

**Expected:**
The name should be set for all controls according to its use.

**User Impact:**
The screen reader users will not get the right reference for that control on what it is used for.

**Attachment:**
